test/unit/client.test.js

```diff
@@ -95,7 +92,8 @@ test('Configure host', t => {
     t.end()
   })
 
-  t.test('Single object', t => {
+  // toDo: modify node roles when master is not supported
```

Comment:
nit: `TODO`, all caps is standard and some extensions/IDEs highlight `TODO`

Reply:
will change from `toDo` to `TODO`